src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const allCommands = sourceString.split(';');
+
+  allCommands.forEach((item) => {
+    const commandParts = item.split(':');
+
+    if (commandParts[0] !== undefined && commandParts[1] !== undefined) {
+      const key = commandParts[0].trim();
+      const value = commandParts[1].trim();
+
+      result[key] = value;
+    }
```

Comment:
let's improve your solution 
```suggestion
    const commandParts = item.split(':');
    const [firstCommandPart, secondCommandPart] = commandParts;

    if (firstCommandPart && secondCommandPart) {
      const key = firstCommandPart.trim();
      const value = secondCommandPart.trim();

      result[key] = value;
    }
```